cpp/include/cudf/quantiles.hpp

```diff
@@ -94,5 +95,31 @@ std::unique_ptr<table> quantiles(
   std::vector<null_order> const& null_precedence = {},
   rmm::mr::device_memory_resource* mr            = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief Calculate approximate percentiles on an input tdigest column.
+ *
+ * tdigest (https://arxiv.org/pdf/1902.04023.pdf) columns are produced specifically
+ * by the TDIGEST and MERGE_TDIGEST aggregations.  These columns represent
+ * compressed representations of a very large input data set that can be
+ * queried for quantile information.
+ *
+ * Produces a LIST column where each row `i` represents output from querying the
+ * corresponding tdigest from `input` row `i`. The length of each output list
+ * is the number of percentages specified in `percentages`.
+ *
+ * @param input           tdigest input data. One tdigest per row.
+ * @param percentiles     Desired percentiles in range [0, 1].
+ * @param output_type     Desired (numeric) output type.
+ * @param mr              Device memory resource used to allocate the returned column's device
+ * memory
+ *
+ * @returns LIST Column containing requested percentile values.
+ */
+std::unique_ptr<column> percentile_approx(
```

Comment:
It seems like we have a nomenclature problem where the other APIs are "quantile" and this is "percentile". 

I've been looking around and the difference seems subtle, and I'm not sure which one we actually mean in this API vs the `quantiles` API. https://stats.stackexchange.com/questions/156778/percentile-vs-quantile-vs-quartile

It _seems_ like the `quantiles` API should really be renamed to `percentiles`.